client/components/Layout/LayoutCollectionHeader.tsx

```diff
@@ -94,13 +95,20 @@ const LayoutCollectionHeader = (props: Props) => {
 		<MetadataDetails collection={collection} content={content} key={3} />,
 	].filter((x) => x);
 
+	// If arcadia, use A-Z byline option
+	const bylineOption =
+		collection.communityId === '1a71ef4d-f6fe-40d3-8379-42fa2141db58'
+			? BylineOption.Az
+			: BylineOption.By;
```

Comment:
I know it's obvious, but could we add a `TODO: implement byline config in scopes` or something along those lines? Same goes for all of the following places we set the byline option for Arcadia.

Reply:
My vote is for moving all of the Arcadia logic inside of `Byline` (likewise, until we systematize this some more).